let User,CustomErrorHandler;_9cc‍.x([["default",()=>_9cc‍.o]]);_9cc‍.w("../../models",[["User",["User"],function(v){User=v}]]);_9cc‍.w("../../services/CustomErrorHandler",[["default",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);


const  userController = {

    async me(req,res,next){
        try{
            const user = await User.findOne({_id:req.user._id}).select('-password -updatedAt -__v');
            if(!user){
                return next(CustomErrorHandler.notFound())
            }
            res.json(user);
        }catch(err){
            return next(err);
        }
    }
};

_9cc‍.d(userController);